### PR TITLE
Music Album and Genre Classes

### PR DESCRIPTION
**Pull request by Chimoneg27**
### For this feature I did the following

- Created the music album and genre files
- Created the music album and genre classes
- Created the initialize methods for the music album and genre classes

### CODE REVIEW

File: classes/genre.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+class Genre
+  attr_accessor :name
+  attr_reader :id, :items
+
+  def initialize(name)
```
Comment (zhorabay) [inline classes/genre.rb:5]: Please, kindly add 'id' to the initialize method. It should take 'id' parameter as well.

**Review by zhorabay — CHANGES_REQUESTED** [review]
Hi @Chimoneg27  💯 💯 💯

Good job so far!

There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

Please, kindly fix linter errors

_Check the comments under the review._

File: classes/genre.rb (hunk 1 above)
Comment (Chimoneg27) [reply classes/genre.rb:5]: @zhorabay The Id and item are private attributes

Comment (Chimoneg27) [reply classes/genre.rb:5]: You can check the UML diagram for reference

**Comment by zhorabay** [comment]
Hi @Chimoneg27 

## Status: APPROVED 🟢

Your project is complete! There is nothing else to say other than... it's time to merge it: ship it:

Congratulations! 🎉 👏 👏 💯

Highlights!
• Followed Gitflow ✔️
• Good commit messages ✔️
• Linters are passing ✔️
• Descriptive Pull request ✔️

**Review by zhorabay — APPROVED** [review]
Hi @Chimoneg27 

## Status: APPROVED 🟢

Your project is complete! There is nothing else to say other than... it's time to merge it: ship it:

Congratulations! 🎉 👏 👏 💯

Highlights!
• Followed Gitflow ✔️
• Good commit messages ✔️
• Linters are passing ✔️
• Descriptive Pull request ✔️

**Pull request merged**